src/test/compile-fail/typeck-builtin-bound-type-parameters.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn foo<T:Copy<U>, U>(x: T) {
+    //~^ ERROR: wrong number of type arguments: expected 0 found 1
```

Comment:
The `ERROR` match doesn't check on span positions. The `^` means the line above (L12) should raise this error. You can just remove the spaces in this `ERROR` line and it'll just work.